packages/react-server/src/ReactFlightServer.js

```diff
@@ -1218,9 +1275,30 @@ function renderModelDestructive(
         );
       }
       case REACT_LAZY_TYPE: {
-        const payload = (value: any)._payload;
-        const init = (value: any)._init;
+        // Reset the task's thenable state before continuing. If there was one, it was
+        // from suspending the lazy before.
+        task.thenableState = null;
```

Comment:
Maybe a silly question, but why adding / changing `debugInfo` has side-effects on `task.thenableState`?
Or is it just a small drive-by fix?

Reply:
Yea, it's mainly a drive-by fix. The bug was introduced in https://github.com/facebook/react/pull/28068